### ISSUE TITLE
Highlighting noteworthy data

### ISSUE THREAD

**Issue by ThomDietrich** [OutsideIT/FireMotD#91]
Hey @willemdh and all,
I just realized that even though FireMotD presents a lot of useful information I actually never "check" it for irregularities, even though that is how we ideally should use it.

Therefore I suggest to introduce a threshold (max AND/OR min) for values. If outside of the threshold, a value will be presented in a theme-specific highlighting style. Wdyt?

Here's a super professional draft:

![grafik](https://user-images.githubusercontent.com/2870104/35002513-3003a486-faea-11e7-8c39-d7127a03573d.png)

**Comment by dimon222** [comment]
I guess if we want to do that, we will have to start collecting historical data, which will be completely different level.

**Comment by willemdh** [comment]
Hey all, this has been on my to do for a long time. Not sure why we would have to collect historical data @dimon222 . We could just set a MaxValue and / or MinValue in the json exportfile and a ValueWarningColor and ValueCriticalColor in the new json templates. Thanks for making the issue.

**Comment by ThomDietrich** [comment]
@dimon222 I agree with Willem. That was not my intention and I think we can leave these areas to more complex tools like Nagios.

@willemdh sure thing. You've mentioned Warning and Critical, that is already more complex than my proposal and I'm not sure if it is needed.
